<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zeal::WidgetUi::MainWindow</class>
 <widget class="QMainWindow" name="Zeal::WidgetUi::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zeal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="resultPane">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="ToolBarFrame" name="searchEditWidget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="searchToolBarFrame">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="SearchEdit" name="lineEdit">
           <property name="placeholderText">
            <string>Enter your query</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="tocSplitter">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QTreeView" name="treeView">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>[&amp;.]Options</string>
    </property>
    <addaction name="actionNewTab"/>
    <addaction name="actionCloseTab"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionDocsets"/>
    <addaction name="actionAboutZeal"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset theme="preferences-desktop">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Prefere&amp;nces</string>
   </property>
  </action>
  <action name="actionAboutZeal">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About &amp;Zeal</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="go-previous">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Back</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="go-next">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Forward</string>
   </property>
  </action>
  <action name="actionNewTab">
   <property name="icon">
    <iconset theme="tab-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New &amp;Tab</string>
   </property>
  </action>
  <action name="actionCloseTab">
   <property name="text">
    <string>&amp;Close Tab</string>
   </property>
  </action>
  <action name="actionNextTab">
   <property name="text">
    <string>Next Tab</string>
   </property>
  </action>
  <action name="actionPreviousTab">
   <property name="text">
    <string>Previous Tab</string>
   </property>
  </action>
  <action name="actionSubmitFeedback">
   <property name="text">
    <string>&amp;Submit Feedback...</string>
   </property>
  </action>
  <action name="actionCheckForUpdates">
   <property name="text">
    <string>&amp;Check for updates...</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset theme="edit-find">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
  </action>
  <action name="actionDocsets">
   <property name="text">
    <string>&amp;Docsets...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SearchEdit</class>
   <extends>QLineEdit</extends>
   <header>ui/widgets/searchedit.h</header>
  </customwidget>
  <customwidget>
   <class>ToolBarFrame</class>
   <extends>QWidget</extends>
   <header>ui/widgets/toolbarframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
